README.md

```diff
@@ -86,12 +86,12 @@ Per default no authentication is needed. To protect api, integrate it with your
 * `GROUP_ACCESS_ONLY`: Force visibility to templates of group only. See also `OIDC_GROUPS_CLAIM`. (default: False)
 * `OIDC_USERINFO_ENDPOINT`: Url of userinfo endpoint as [described](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo)
 * `OIDC_GROUPS_CLAIM`: Name of claim to be used to define group membership (default: document_merge_service_groups)
+* `OIDC_BEARER_TOKEN_REVALIDATION_TIME`: Time in seconds before bearer token validity is verified again. Disabled per default for best security, but might be helpful in case of slow Open ID Connect provider. It uses [cache](#cache) mechanism for memorizing userinfo result. Number has to be lower than access token expiration time. (default: 0)
```

Comment:
Instead of "disabled by default", I'd change the text to say "A value of 0 effectively disables the cache"

Reply:
this is misleading indeed. I tried to rephrase a bit differently.